test/visibility/import/deeperNesting-parentImport.chpl

```diff
@@ -0,0 +1,18 @@
+module A {
+  module B {
+    proc foo() {
+      writeln("In A.B.foo()");
+    }
+
+    var x: int;
+
+    module C {
+      import A.B; // Should work
+
+      proc main() {
+        writeln(B.x);
+        B.foo();
```

Comment:
Can you add a copy of this test that checks that we don't find a bare `foo();` call (without the `B.`) ?

Reply:
That behaves identically to this program:
```chapel
module A {
  module B {
    proc foo() {
      writeln("In A.B.foo()");
    }

    var x: int;

    module C {
      proc main() {
        foo();
      }
    }
  }
}
```
which is to say, that we can call `foo()` without qualification, due to it being defined in an outer scope.

Reply:
I was looking for a test to make sure `import X.Y` didn't accidentally behave equivalent to `use`. Is there some other pattern you can think of to do that?

Reply:
Ah, like:
```chapel
module A {
  module B {
    module C {
      proc foo() {
        writeln("In A.B.C.foo()");
      }

      var x: int;
    }
  }
}

module User {
  import A.B.C;

  proc main() {
    writeln(x);
    foo();
  }
}
```
(but I'll split them up, since we can't get both error messages in the same output)

Reply:
sounds good